### PR TITLE
Add blog post for launch, inc. migration details

### PR DESCRIPTION
**Pull request by elisechant**

### CODE REVIEW

File: src/posts/gold-design-system.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+### New docs site
+
+Docs are available at [https://gold.designsystemau.org/](https://gold.designsystemau.org/), which should now be used instead of the DTA's decommissioned docs site.
+ 
+The code can be found at [Github](https://github.com/designsystemau/gold-design-system-site).
+ 
+
+### New npm packages
+
+Npm packages are available at the new [@gold.au organisation](https://www.npmjs.com/org/gold.au).
```
Comment (elisechant) [inline src/posts/gold-design-system.md:48]: include a screenshot

File: src/posts/gold-design-system.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+There was agreement that the name change was an appropriate identity change marking the end of DTA's custodianship of the project.
+
+In addition, from the RFC:
+
+> having a fun name (as opposed to a plain language name) has the effect of "creating shared language" and is better to "excite people about working together, "particularly non-technical people. That this is beneficial for "creating unity and delight" and gives your people an identity.
+
+ 
+### New docs site
+
```
Comment (elisechant) [inline src/posts/gold-design-system.md:40]: include a screenshot

File: src/posts/gold-design-system.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
+---
+title: Celebrating the launch of GOLD Design System 🚀
+date: '2021-09-21'
+tags: [announcements, releases]
+description: GOLD Design System has been published and available to be used in place of the Australian Government Design System   
+permalink: posts/{{ title | slug }}/index.html
+---
+
+Today marks the launch of GOLD Design System, replacing the Australian Government Design System.
```
Comment (dominikwilkowski) [inline src/posts/gold-design-system.md:9]: Should that be "the GOLD design system" as in "the government open language for design"?

File: src/posts/gold-design-system.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+To continue to benefit from future updates to the Design System, including security patches, see bottom of page for [Migration details](#migration-details). 
+ 
+### Reasons for changing the name to GOLD
+ 
+Some might remember GOLD was the original name of the design system - the Government Open Language of Design.
+
+<img src="/images/previous-gold-docs.png" style="box-shadow: 1px 1px 10px #cccccc; border: 1px solid #e5e5e5;" />
+
+As the curatorship of Australian Government Design System by the Digital Transformation Agency ends, we thought it would be appropriate to bring GOLD back!
```
Comment (dominikwilkowski) [inline src/posts/gold-design-system.md:23]: I like it. Maybe without the explanation point to not put a finer point on it :D haha

**Review by dominikwilkowski — COMMENTED** [review]
Love it :) 
Added a question or two.

File: src/posts/gold-design-system.md (hunk 3 above)
Comment (elisechant) [reply src/posts/gold-design-system.md]: Included the suggestion, plus the next paragraph.

**Comment by elisechant** [comment]
@dominikwilkowski are you comfortable with the migration documentation being here, and/or should we also add this elsewhere? As in close to code?

**Comment by dominikwilkowski** [comment]
I think it's great that it's here. Relevant in this context.

**Pull request merged**